<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>676</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>920</width>
    <height>676</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>920</width>
    <height>676</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string extracomment="aaa"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="img_window" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>612</width>
      <height>512</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>306</width>
      <height>256</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>612</width>
      <height>512</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="message_box">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>430</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="PEAR_NO_LABEL">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>13</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Pear #</string>
    </property>
   </widget>
   <widget class="QLabel" name="pear_no_label">
    <property name="geometry">
     <rect>
      <x>809</x>
      <y>13</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="dec_pear_no_btn">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>13</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inc_pear_no_btn">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>13</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="SET_BRT_LABEL">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>調光値設定</string>
    </property>
   </widget>
   <widget class="Line" name="vline">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>20</width>
      <height>651</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="hline2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>160</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="hline1">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="hline3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>280</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="save_img_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>620</y>
      <width>531</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>検査の実行</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="hline4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>390</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="LED_NO_LABEL">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>90</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>LED #</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="chsel_combobox">
    <property name="geometry">
     <rect>
      <x>751</x>
      <y>90</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="BRIGHTNESS_LABEL">
    <property name="geometry">
     <rect>
      <x>681</x>
      <y>120</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>調光値
 (0-255)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="brightness_spinbox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>127</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="OP_DATA_LABEL">
    <property name="geometry">
     <rect>
      <x>672</x>
      <y>330</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>運転データ</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="opdata_combobox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>330</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentText">
     <string>1</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="inc_rot_x1_btn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>550</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inc_rot_x3_btn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>550</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="inc_rot_x2_btn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>550</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zhome_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>550</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ZHOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dec_rot_x3_btn">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dec_rot_x2_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>550</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dec_rot_x1_btn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>550</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="OP_MOTOR_LABEL_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>400</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ログ</string>
    </property>
   </widget>
   <widget class="QLabel" name="TABLE_ROTATION_LABEL">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>370</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>現在の回転角 [deg]:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="turntable_rot_label">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>370</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Meiryo UI</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <zorder>img_window</zorder>
   <zorder>LED_NO_LABEL</zorder>
   <zorder>chsel_combobox</zorder>
   <zorder>BRIGHTNESS_LABEL</zorder>
   <zorder>brightness_spinbox</zorder>
   <zorder>OP_DATA_LABEL</zorder>
   <zorder>opdata_combobox</zorder>
   <zorder>message_box</zorder>
   <zorder>PEAR_NO_LABEL</zorder>
   <zorder>pear_no_label</zorder>
   <zorder>dec_pear_no_btn</zorder>
   <zorder>inc_pear_no_btn</zorder>
   <zorder>vline</zorder>
   <zorder>hline2</zorder>
   <zorder>SET_BRT_LABEL</zorder>
   <zorder>hline1</zorder>
   <zorder>SET_LED_LABEL</zorder>
   <zorder>hline3</zorder>
   <zorder>apply_led_btn</zorder>
   <zorder>apply_brt_btn</zorder>
   <zorder>save_img_btn</zorder>
   <zorder>hline4</zorder>
   <zorder>inc_rot_x1_btn</zorder>
   <zorder>inc_rot_x3_btn</zorder>
   <zorder>inc_rot_x2_btn</zorder>
   <zorder>zhome_btn</zorder>
   <zorder>dec_rot_x3_btn</zorder>
   <zorder>dec_rot_x2_btn</zorder>
   <zorder>dec_rot_x1_btn</zorder>
   <zorder>TABLE_ROTATION_LABEL</zorder>
   <zorder>turntable_rot_label</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
